include/CppUTest/CppUTestConfig.h

```diff
@@ -201,10 +201,15 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Handling of systems with a different int-width (e.g. 16 bit).
+ */
+#if defined(__WATCOMC__)
+#define CPPUTEST_16BIT_INTS
+#endif
```

Comment:
I'd prefer the previous solution and not having Watcom checks in this header (when we can avoid them)